File: Baekjoon/구현/15820 맞았는데 왜 틀리죠?/Inqbator.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;

public class Inqbator {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int[] input = Arrays.stream(br.readLine().split(" ")).mapToInt(it -> Integer.parseInt(it)).toArray();

        for (int i = 0; i < input[0]; i++) {
            int[] answerAndTestCase = Arrays.stream(br.readLine().split(" ")).mapToInt(it -> Integer.parseInt(it)).toArray();
            if (answerAndTestCase[0] != answerAndTestCase[1]) {
                System.out.println("Wrong Answer");
                return;
            }
        }

        for (int i = 0; i < input[1]; i++) {
            int[] answerAndTestCase = Arrays.stream(br.readLine().split(" ")).mapToInt(it -> Integer.parseInt(it)).toArray();
            if (answerAndTestCase[0] != answerAndTestCase[1]) {
                System.out.println("Why Wrong!!!");
                return;
            }
        }

        System.out.println("Accepted");
    }
}

File: Baekjoon/구현/15820 맞았는데 왜 틀리죠?/Inqbator.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val (cntOfSampleTest, cntOfPrivateTest) = readLine().split(" ").map {it.toInt()}
    fun checkIsAllCorrect(cntOfCase: Int): Boolean {
        repeat(cntOfCase) {
            val (myAnswer, realAnswer) = readLine().split(" ").map { it.toInt() }
            if (myAnswer != realAnswer) {
                return false
            }
        }
        return true
    }

    val outputMsg = if (!checkIsAllCorrect(cntOfSampleTest))
        "Wrong Answer"
    else if (!checkIsAllCorrect(cntOfPrivateTest))
        "Why Wrong!!!"
    else
        "Accepted"

    println(outputMsg)
}